common/util/iterator_range.h

```diff
@@ -46,6 +46,8 @@ class iterator_range {
   const iterator& begin() const { return begin_; }
   const iterator& end() const { return end_; }
 
+  size_t size() { return end_ - begin_; }
+
```

Comment:
don't add `size()` to iterator_range, because it should reflect existing minimalist iterator_range libraries that exist out there.  Instead, use the `util/container_iterator_range` we've provided (and could be enhanced as we wish).

Reply:
Sure, changed the code where I used to avoid that. Thanks